service/types.go

```diff
@@ -305,3 +307,38 @@ type signedTransactionWrapper struct {
 	SignedTransaction []byte          `json:"signed_tx"`
 	Currency          *types.Currency `json:"currency,omitempty"`
 }
+
+type signedTransactionWrapperWire struct {
+	SignedTransaction []byte          `json:"signed_tx"`
+	Currency          *types.Currency `json:"currency,omitempty"`
+}
+
+func (t *signedTransactionWrapper) MarshalJSON() ([]byte, error) {
```

Comment:
You don't need to override `MarshalJSON` because it doesn't change default behavior.